src/theory/logic_info.cpp

```diff
@@ -412,6 +413,11 @@ void LogicInfo::setLogicString(std::string logicString)
     } else {
       enableQuantifiers();
     }
+    if (!strncmp(p, "SEP_", 3))
```

Comment:
```suggestion
    if (!strncmp(p, "SEP_", 4))
```

Otherwise, we'd match `"SEPx"`